nix/checks/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  hello,
+  topiaryPkgs,
+  pre-commit-hook,
+}:
+
+let
+  inherit (builtins) deepSeq;
+
+in
+{
+  # NOTE: The following checks have been removed as WASM
+  # and playground development has moved to the playground branch:
+  # - clippy-wasm
+  # - topiary-playground
+  inherit (topiaryPkgs)
+    clippy
+    fmt
+    topiary-core
+    audit
+    benchmark
+    topiary-cli
+    ;
+
+  ## Check that the `lib.pre-commit-hook` output builds/evaluates
+  ## correctly. `deepSeq e1 e2` evaluates `e1` strictly in depth before
+  ## returning `e2`. We use this trick because checks need to be
+  ## derivations, which `lib.pre-commit-hook` is not.
```

Comment:
Why the double `#`?

Reply:
Ah, sorry, force of habit, single `#` for when it's meant to be uncommented, double otherwise. It's true that the Nix ecosystem uses only single hashes. (At least `nixfmt-rfc-style` doesn't like them.) Will change that.

Reply:
Done in b853c7827d056b26ae69f52e267f8549ca2a2a9c.